src/test/index.ts

```diff
@@ -139,3 +133,80 @@ describe('Error test', function () {
     expect(emailError.code).to.equal(400);
   });
 });
+describe('Login test', function () {
+  it('Should check the login setup', async () => {
```

Comment:
Não entendi muito bem o que quer dizer "should check the login setup". O nome do teste deveria descrever bem o que ele testa.